linkerd/docs/config.md

```diff
@@ -59,19 +39,27 @@ tracers:
   sampleRate: 0.02
 ```
 
-The configuration may be specified as a JSON or YAML object, as described
-below.  Four top level keys are supported:
+Welcome to the Configuration Reference for linkerd!
+
+linkerd's configuration is controlled via config file, which must be provided
+as a command-line argument. It may be a local file path or `-` to
+indicate that the configuration should be read from the standard input.
+For convenience, the release package includes a default `linkerd.yaml` file in
+the `config/` directory.
+
+#### File Format
+
+The configuration may be specified as a JSON or YAML object. There are no requirements on field ordering, though it's generally good style to start a router with the _protocol_. Four top level keys are supported:
 
-* [Admin](#admin)
-* [Routers](#routers)
-* [Namers](#namers)
-* [Tracers](#tracers)
+Key | Required? | Value Description
```

Comment:
TIOLI: I would drop the `?` from `Required?` and drop the `Value` from `Value Description`